src/others/xml/parsers/lv_xml_label_parser.c

```diff
@@ -58,9 +58,10 @@ void lv_xml_label_apply(lv_xml_parser_state_t * state, const char ** attrs)
         if(lv_streq("text", name)) lv_label_set_text(item, value);
         else if(lv_streq("long_mode", name)) lv_label_set_long_mode(item, long_mode_text_to_enum_value(value));
 #if LV_USE_TRANSLATION
-        if(lv_streq("text-translated", name)) lv_label_set_text(item, lv_tr(value));
+        else if(lv_streq("text-translated", name)) lv_label_set_text(item, lv_tr(value));
+        else if(lv_streq("translation-tag", name)) lv_label_set_translation_tag(item, value);
```

Comment:
```suggestion
        else if(lv_streq("translation_tag", name)) lv_label_set_translation_tag(item, value);
```
Normally we should use the property names exactly as they are in the LVGL API (`translation_tag` instead of `translation-tag`).

`text-translated` is not needed anymore.

Reply:
Removed!